src/FixedSizeArrays.jl

```diff
@@ -23,13 +23,17 @@ const FixedSizeMatrix{T} = FixedSizeArray{T,2}
 function FixedSizeArray{T,N}(::UndefInitializer, size::NTuple{N,Int}) where {T,N}
     FixedSizeArray{T,N}(Internal(), Memory{T}(undef, checked_dims(size)), size)
 end
-function FixedSizeArray{T,N}(::UndefInitializer, size::Vararg{Int,N}) where {T,N}
+function FixedSizeArray{T,N}(::UndefInitializer, size::NTuple{N,Integer}) where {T,N}
+    ints = map(Int, size)::NTuple{N,Int}  # prevent infinite recursion
```

Comment:
Probably not too important, but I'm not sure if it's better to use the `Int` constructor or `convert`, like `convert(Int, x)`.

Reply:
`convert(Int, x)` should have an extra type assertion compared to plain `Int(x)`, which may be good, but you're also adding the `::NTuple{N,Int}` assertion here.